sagemaker-python-sdk/tensorflow_distributed_mnist/tensorflow_batch_transform_mnist.ipynb

```diff
@@ -156,55 +158,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Inspect input data\n",
+    "# Perform inference\n",
     "\n",
-    "Before we perform the inference let's inspect the input data."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import boto3\n",
-    "import matplotlib.pyplot as plt\n",
-    "import os\n",
-    "from numpy import genfromtxt\n",
-    "\n",
-    "plt.rcParams[\"figure.figsize\"] = (2,10)\n",
-    "\n",
-    "def show_digit(img, caption='', subplot=None):\n",
-    "    if subplot==None:\n",
-    "        _,(subplot)=plt.subplots(1,1)\n",
-    "    imgr=img.reshape((28,28))\n",
-    "    subplot.axis('off')\n",
-    "    subplot.imshow(imgr, cmap='gray')\n",
-    "    plt.title(caption)\n",
-    "\n",
-    "tmp_dir = '/tmp/data'\n",
-    "input_file_name = 'data.csv'\n",
-    "input_file_path = 'batch-transform/mnist/' + input_file_name\n",
-    "input_bucket_name = 'sagemaker-sample-data-{}'.format(boto3.Session().region_name)\n",
-    "\n",
-    "if not os.path.exists(tmp_dir):\n",
-    "    os.makedirs(tmp_dir)\n",
-    "\n",
-    "s3 = boto3.resource('s3')\n",
-    "\n",
-    "s3.Bucket(input_bucket_name).download_file(input_file_path, os.path.join(tmp_dir, input_file_name))\n",
-    "input_data = genfromtxt(os.path.join(tmp_dir, input_file_name), delimiter=',')\n",
-    "\n",
-    "show_digit(input_data)"
+    "Now that we've trained a model, we're going to use it to perform inference with a SageMaker batch transform job. The request handling behavior of the Endpoint deployed during the transform job is determined by the `mnist.py` script we looked at earlier."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Running a batch transform job\n",
+    "## Run a batch transform job\n",
     "\n",
-    "Now we can perform some inference with the model we've trained by running a batch transform job. The request handling behavior of the Endpoint deployed during the transform job is determined by the `mnist.py` script."
+    "For our batch transform job, we're going to use input data that contains ten MNIST images, located in the public SageMaker sample data S3 bucket. To create the batch transform job, we simply call `transform()` on our transformer with information about the input data."
```

Comment:
Is there a reason we're using such a small number of images for batch transform?